### ISSUE TITLE
Duplicated object spread helpers when using @babel/preset-env with useBuiltins: 'usage'

### ISSUE THREAD

**Issue by fnlctrl** [babel/babel#7693]
Choose one: is this a bug report or feature request?
Bug

### Input Code

```js
// 'a.js'
let a = (obj) => ({...obj})
// 'b.js'
let a = (obj) => ({...obj})
// ....and other files that uses object spread
```

### Babel/Babylon Configuration (.babelrc, package.json, cli command)

```js
{
  presets: [
    ['@babel/preset-env', {
      modules: false,
      useBuiltIns: 'usage',
      targets: { android: '4.4', ios: '9' }
    }]
  ]
}
```

### Expected Behavior
Only one occurrence of object spread helpers.

### Current Behavior
Duplicated `function _objectSpread() {...}` and `function _defineProperty(obj, key, value) {...}` per file 

### Possible Solution
Maybe: 
1. Make [babel-plugin-proposal-object-rest-spread](https://github.com/babel/babel/tree/master/packages/babel-plugin-proposal-object-rest-spread) support `useBuiltins: 'usage'` as it currently supports only true/false, or treat `'usage'` as `true`.
2. Or allow passing options to individual plugins inside option of `preset-env`, something like
```
    ['@babel/preset-env', {
      pluginOptions: {
         '@babel/plugin-proposal-object-rest-spread': {
           useBuiltIns: 'true'
         }
      }
    ]
```

### Context

I want to use the new `useBuiltIns: 'usage'` feature to polyfill on demand, making smaller outputs without manually defining `exclude` option for ES6/7 features that I don't actually use. However this is instead duplicated object spread helpers that bloats the output.

### Your Environment
| software         | version(s)
| ---------------- | -------
| Babel            |  7.0.0.beta-44
| node             | 9.10.1
| npm              | 5.6.0
| Operating System |  Windows 10 x64

**Comment by fnlctrl** [comment]
I was actually using `preset-stage-2` with `preset-env`
```js
presets: [ '@babel/preset-stage-2',  '@babel/preset-env']
```
and removing `preset-stage-2` gives me warnings that I should add `@babel/plugin-proposal-object-rest-spread` to plugins. 

So I added `@babel/plugin-proposal-object-rest-spread`  to plugins with `{useBuiltIns: true}` option, but it still gives bloated output.

Then I tried removing `preset-env` and it solved this problem. So I guess the real issue is probably `preset-env` overriding the  `{useBuiltIns: true}`  option I passed to the plugin.
